oci/runtime_oci.go

```diff
@@ -361,7 +361,7 @@ func (r *runtimeOCI) ExecSyncContainer(c *Container, command []string, timeout i
 		args = append(args, fmt.Sprintf("%d", timeout))
 	}
 	args = append(args, "-l", logPath)
-	args = append(args, "--socket-dir-path", ContainerAttachSocketDir)
+	args = append(args, "--socket-dir-path", r.containerAttachSocketDir)
```

Comment:
Should this be config. instead of r.containreAttachSockerDir here?  I'm not sure, so thought I'd ask.

Reply:
Hm, it should be the same since the runtime will be created with `oci.New()` which receives the `config.ContainerAttachSocketDir` parameter.

A further optimization for the future would be to pass the config pointer directly into the OCI runtime to avoid copying the variables around.